packages/block-library/src/latest-posts/edit.js

```diff
@@ -347,6 +359,10 @@ class LatestPostsEdit extends Component {
 							'trim',
 						] );
 						let excerpt = post.excerpt.rendered;
+						const currentAuthor = find(
+							authors,
+							( author ) => author.id === post.author
+						);
```

Comment:
You don't have to use Lodash:
```suggestion
						const currentAuthor = authors.find(
							( author ) => author.id === post.author
						);
```